Vagrantfile

```diff
@@ -0,0 +1,32 @@
+# set up a Vagrantbox to run the repl
+# check it out at http://phpepl.dev after running 'vagrant up'
```

Comment:
Also `vagrant plugin install vagrant-hostsupdater` to get the hostname automatically in /etc/hosts or whatever the host equivalent.